### ISSUE TITLE
I have a question

### ISSUE THREAD

**Issue by YZF-Myself** [vishal3477/Reverse_Engineering_GMs#4]
hello, do i need to create all the paths in the reverse_eng.py ? what do i need to save for wach folder?

parser.add_argument('--lr', default=0.0001, type=float, help='learning rate')
parser.add_argument('--data_train',default='mnt/scratch/asnanivi/GAN_data_6/set_1/train',help='root directory for training data')
parser.add_argument('--data_test',default='mnt/scratch/asnanivi/GAN_data_6/set_1/test',help='root directory for testing data')
parser.add_argument('--ground_truth_dir',default='./',help='directory for ground truth')
parser.add_argument('--seed', default=1, type=int, help='manual seed')
parser.add_argument('--batch_size', default=16, type=int, help='batch size')
parser.add_argument('--savedir', default='/mnt/scratch/asnanivi/runs')
parser.add_argument('--model_dir', default='./models')
parser.add_argument('--N_given', nargs='+', help='position number of GM from list of GMs used in testing', default=[1,2,3,4,5,6])

os.chmod('./mnt/scratch',0o777)
os.makedirs('.%s/result_3/%s' % (save_dir, sig), exist_ok=True)

i also had a mistake:Couldn't find any class folder in mnt/scratch/asnanivi/GAN_data_6/set_1/train

Thanks!

**Comment by vishal3477** [comment]
Hi, 
So for the testing of GMs, you have to move the GMs from training to testing folder. We tested on 6 different test sets. Please have a look in the paper's supplementary for the details.
